npm/private/npm_import.bzl

```diff
@@ -790,6 +790,10 @@ def _impl_links(rctx):
         virtual_store_root = utils.virtual_store_root,
         maybe_bins = maybe_bins,
         dev = rctx.attr.dev,
+        # Insert nothing when `lifecycle_hooks_use_default_shell_env` is None to remain backwards-compatible
+        # with bazel-lib `run_binary` before `use_execroot_entry_point` was added.
+        # TODO(2.0): remove support for old bazel-lib without `run_binary(use_execroot_entry_point)`
```

Comment:
todo looks wrong, use_execroot_entry_point is copy-pasta?

Reply:
Yes that's wrong, there's a lot of copy-pasta duplicating all the attribute passthru everywhere :/